### PR TITLE
test: add orchestrator tests

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Depends on https://github.com/celestiaorg/orchestrator-relayer/pull/133 and https://github.com/celestiaorg/orchestrator-relayer/pull/115

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/51

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: testing/qgb.go [hunk 1]
```diff
@@ -13,3 +21,19 @@ func NewEVMClient(key *ecdsa.PrivateKey) *evm.Client {
 	// the simulated backend doesn't provide an RPC endpoint.
 	return evm.NewClient(logger, nil, key, "", 100000000)
 }
+
+func NewOrchestrator(
+	node *TestNode,
+) *orchestrator.Orchestrator {
```
Comment (evan-forbes) [inline testing/qgb.go:27]: [non blocking nit]
why format this like this?

File: testing/qgb.go [hunk 2]
```diff
@@ -13,3 +21,19 @@ func NewEVMClient(key *ecdsa.PrivateKey) *evm.Client {
...
+	logger := tmlog.NewNopLogger()
+	appQuerier := rpc.NewAppQuerier(logger, node.CelestiaNetwork.GRPCClient, encoding.MakeConfig(app.ModuleEncodingRegisters...))
+	tmQuerier := rpc.NewTmQuerier(node.CelestiaNetwork.Client, logger)
+	p2pQuerier := p2p.NewQuerier(node.DHTNetwork.DHTs[0], logger)
+	broadcaster := orchestrator.NewBroadcaster(node.DHTNetwork.DHTs[0])
+	retrier := orchestrator.NewRetrier(logger, 3, 500*time.Millisecond)
+	orch, err := orchestrator.New(logger, appQuerier, tmQuerier, p2pQuerier, broadcaster, retrier, *celestiatestnode.NodeEVMPrivateKey)
+	if err != nil {
+		panic(err)
+	}
```
Comment (evan-forbes) [inline testing/qgb.go:37]: [optional]
why not pass a testing.T as an arg to handle errors? are we using this function outside of tests?

**Review by evan-forbes — COMMENTED** [review]
a few minor comments but otherwise LGTM

**Review by evan-forbes — APPROVED** [review]

File: testing/qgb.go (hunk 1 above)
Comment (SweeXordious) [reply testing/qgb.go:27]: no reason :D just happened to be like that.  should I change?

File: testing/qgb.go (hunk 2 above)
Comment (SweeXordious) [reply testing/qgb.go:37]: https://github.com/celestiaorg/orchestrator-relayer/issues/153

File: testing/qgb.go (hunk 1 above)
Comment (SweeXordious) [reply testing/qgb.go:27]: @evan-forbes Can I merge or I should make the change?

Comment (evan-forbes) [reply testing/qgb.go:27]: up to you! if we don't have linters for it, then I don't think its worth blocking on.

Comment (SweeXordious) [reply testing/qgb.go:27]: Cool, thanks

**Pull request merged**